app/_src/gateway/support/index.md

```diff
@@ -3,7 +3,7 @@ title: Kong Gateway Enterprise Version Support
 breadcrumb: Distributions
 ---
 
-Kong primarily follows [semantic versioning](https://semver.org/) (SemVer) with its products, with an added segment for {{site.ee_product_name}} patch releases. Products follow a pattern of `{MAJOR}.{MINOR}.{PATCH}.{ENTERPRISE_PATCH}`. The `ENTERPRISE_PATCH` segment identifies a sub-patch based on the Kong Community Gateway. 
+Kong follows a soft [semantic versioning](https://semver.org/) (SemVer) model with its products, with an added segment for {{site.ee_product_name}} patch releases. Products follow a pattern of `{MAJOR}.{MINOR}.{PATCH}.{ENTERPRISE_PATCH}`. The `ENTERPRISE_PATCH` segment identifies a sub-patch based on the Kong Community Gateway. 
```

Comment:
Wasn't sure how to rephrase this; it was noted in thread that we need to pull back our semver assertions somewhat, but I wasn't sure where that line is.